### ISSUE TITLE
Add parsing for EntityInstances

### ISSUE THREAD

**Issue by estivate** [estivate/ldtk_rust#2]
EntityInstances can be implemented for the most part, but LDtk let's you create arbitrary fields to assign various values to Entities which is going to be an extra step in a strongly typed language like Rust. Specifically: 

```json
{
  "fieldInstances":[
    {
      "__identifier":"text",
      "__value":"Can be arbitrary text...",
      "__type":"String",
      "defUid":88,
    },
    {
      "__identifier":"colors",
      "__value":["#FFD500", "#FFD501", "#FFD502"],
      "__type":"Array<Color>",
      "defUid":90,
    }
  ]
}
```

Not sure how to deal with this in Serde. I can tag it with the serde type "Value" and let the caller figure it out, or maybe I can set up an Enum to use the internally tagged __type field to pick the right option. The possible types are limiited.

**Comment by estivate** [comment]
added EntityInstaces and supporting structs, and for the "__value" field in FieldInstances just used serde::Value for now. Matching on an enum didn't seem super helpful, but we'll see what it's like to program against.

**Comment by estivate** [comment]
published in 0.2.0
